src/Microsoft.DotNet.Darc/src/Darc/Operations/VirtualMonoRepo/ScanOperation.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.DotNet.Darc.Models.VirtualMonoRepo;
```

Comment:
I think there's a missing header. Also usings are usually sorted by System. first and then the rest. Weird that your VS didn't pick up on the `.editorconfig` file

Reply:
I added the license headers. Also sorted the usings